main.go

```diff
@@ -119,6 +126,16 @@ func main() {
 	stringFlagOrEnv(&neuronMonitorImage, "neuron-monitor-image", "RELATED_IMAGE_NEURON_MONITOR", fmt.Sprintf("%s:%s", neuronMonitorImageRepository, v.NeuronMonitor), "The default Neuron monitor image. This image is used when no image is specified in the CustomResource.")
 	pflag.Parse()
 
+	// set instrumentation cpu and memory limits in environment variables to be used for default instrumentation
+	autoInstrumentationConfig := map[string]map[string]string{"java": {"cpu": "500m", "memory": "64Mi"}, "python": {"cpu": "500m", "memory": "32Mi"}, "dotnet": {"cpu": "500m", "memory": "128Mi"}}
+	err := json.Unmarshal([]byte(autoInstrumentationConfigStr), &autoInstrumentationConfig)
+	if err != nil {
+		setupLog.Error(err, "Unable to unmarshal auto-instrumentation config, assuming default values")
```

Comment:
Im not sure if this is an Error if its an optional arg

Reply:
Yeah I don't see this an error, maybe it can be an info saying we are using default values and then print out what those are.

Ideally this shouldn't happen because the defaults should be set in the helm chart, but this is prob going to be used for testing.

Reply:
Is this properly addressed now?